.env.example

```diff
@@ -0,0 +1,4 @@
+# THIRDWEB ENGINE REQUIRED
+ADMIN_WALLET_ADDRESS=<YOUR_ADMIN_WALLET_ADDRESS>
+THIRDWEB_API_SECRET_KEY=<YOUR_SECRET_KEY>
+ENCRYPTION_PASSWORD=<YOUR_ENCRYPTION_PASSWORD>
```

Comment:
Missing, `POSTGRES_CONNECTION_URL`

Reply:
Another env variable missing is `ENABLE_HTTPS`. This allows users to access their engine instance to connect to dashboard.

Reply:
Hey @farhanW3 I added it directly to the docker compose.

Reply:
I added the following ENV as per the docs.